lib/Backends/CPU/libjit/libjit.cpp

```diff
@@ -725,7 +726,17 @@ static void libjit_arg_max_generic(const T *inW, T2 *outW, const dim_t *inWdims,
     for (b = 0; b < inWdims[(axis + 2) % 4]; b++) {
       for (c = 0; c < inWdims[(axis + 3) % 4]; c++) {
 
-        T max = inW[libjit_getXYZW(inWdims, *dim[0], *dim[1], *dim[2], 0)];
+        T max;
+        if (axis == 0) {
+          max = inW[libjit_getXYZW(inWdims, 0, *dim[1], *dim[2], *dim[3])];
+        } else if (axis == 1) {
+          max = inW[libjit_getXYZW(inWdims, *dim[0], 0, *dim[2], *dim[3])];
+        } else if (axis == 2) {
+          max = inW[libjit_getXYZW(inWdims, *dim[0], *dim[1], 0, *dim[3])];
+        } else {
+          max = inW[libjit_getXYZW(inWdims, *dim[0], *dim[1], *dim[2], 0)];
+        }
```

Comment:
nit: This works/I'm fine with it -- but it would probably be simpler to just set `T max = std::numeric_limits<T>::min()`.

Reply:
Done.